### DISCUSSION TITLE
Missing "Edit Script" in Tasmota 12.3.1

### DISCUSSION THREAD

**Discussion by Duffy987** [arendst/Tasmota#17639] [Q&A]
Since Tasmota 12.3.1 I can no longer find the "Edit Script" button in the console, it seems to have disappeared.
Where can I paste the scripts in this version?

Regards

**Comment by sfromis** [comment]
You may have updated to the wrong binary, and need another build including the scripting language feature to have that.

**Reply by Duffy987** [reply]
Thank you
